src/loaders/ncbi_downloader/gtdb.py

```diff
@@ -0,0 +1,311 @@
+"""
+usage: ncbi_downloader.py [-h] --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...] --release_ver {202,207,214,80,83,86,89,95}
```

Comment:
description and usage probably also needs to be updated accordingly.

Reply:
Updated!